### PR TITLE
 Fix Couchbase sudden death in GUI install. Fix Multiple bug report in gui installation paths and add gunicorn

### PR DESCRIPTION
**Pull request by rizarsyi**

### CODE REVIEW

**Comment by rizarsyi** [comment]
include fix #165

File: pygluu/kubernetes/gui/app.py [hunk 1]
```diff
@@ -64,30 +96,26 @@ def main():
...
+    logging.getLogger('werkzeug').disabled = True
     logging.getLogger('socketio').setLevel(logging.ERROR)
     logging.getLogger('engineio').setLevel(logging.ERROR)
-    log.disabled = True
 
-    app.run(host=host, port=port, debug=debug)
+    options = {
+        'bind': '%s:%s' % (args.host, args.port),
+        'worker_class': 'gevent',
+        "debug": args.debug
```
Comment (iromli) [inline pygluu/kubernetes/gui/app.py:115]: I can't find reference to `debug` option in Gunicorn config. Can you share the reference?

Comment (rizarsyi) [reply pygluu/kubernetes/gui/app.py:115]: @iromli https://docs.gunicorn.org/en/0.17.0/configure.html#debugging

Comment (rizarsyi) [reply pygluu/kubernetes/gui/app.py:115]: @iromli i just realized it's an old doc.. :disappointed_relieved:

**Review by moabu — APPROVED** [review]

**Pull request merged**